README.md

```diff
@@ -8,6 +8,11 @@
 <br />
 
 <div align="center">
+   <!-- CI status -->
+  <a href="https://github.com/async-rs/async-std/actions">
+    <img src="https://github.com/async-rs/async-std/workflows/CI/badge.svg"
```

Comment:
It would be nice if we could choose `?style=flat-square` here but I don't think GitHub's badges have configurable style. Maybe I'm wrong though.

Reply:
Currently, GitHub cant customize style. Maybe wait https://github.com/badges/shields/issues/2574.